sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_connector_costs_v1/query.sql

```diff
@@ -25,44 +25,66 @@ total_mar AS (
     destination_id,
     measured_month
 ),
-monthly_connector_costs AS (
+total_mar AS (
   SELECT
-    destinations.destination_name,
+    measured_month,
+    SUM(paid_active_rows) AS paid_active_rows,
+  FROM
+    destinations_total_mar
+  GROUP BY
+    measured_month
+),
+monthly_connector_ratios AS (
+  SELECT
+    destination_id,
     measured_month,
     connector_mar.connector_name,
     connector_mar.paid_active_rows,
     connector_mar.free_active_rows,
     connector_mar.total_active_rows,
+    ROUND(
+      connector_mar.paid_active_rows / NULLIF(destinations_total_mar.paid_active_rows, 0) * 100,
+      2
+    ) AS percentage_of_destination_paid_active_rows,
     ROUND(
       connector_mar.paid_active_rows / NULLIF(total_mar.paid_active_rows, 0) * 100,
       2
-    ) AS percentage_of_total_paid_mar,
+    ) AS percentage_of_total_active_rows
+  FROM
+    connector_mar
+  LEFT JOIN
+    destinations_total_mar
+  USING
+    (destination_id, measured_month)
+  LEFT JOIN
+    total_mar
+  USING
+    (measured_month)
+),
+monthly_connector_costs AS (
+  SELECT
+    destinations.destination_name,
+    measured_month,
+    monthly_connector_ratios.connector_name,
+    monthly_connector_ratios.paid_active_rows,
+    monthly_connector_ratios.free_active_rows,
+    monthly_connector_ratios.total_active_rows,
+    monthly_connector_ratios.percentage_of_destination_paid_active_rows,
+    monthly_connector_ratios.percentage_of_total_active_rows,
     ROUND(
-      connector_mar.paid_active_rows / NULLIF(
-        total_mar.paid_active_rows,
-        0
-      ) * monthly_costs.dollars_spent,
+      monthly_connector_ratios.percentage_of_destination_paid_active_rows * monthly_costs.dollars_spent,
```

Comment:
For this to be accurate you'd have to divide the percentage by 100 first.

However, I'd suggest having just the ratios calculated in the `monthly_connector_ratios` CTE, then you can use them here as is, and multiply them by 100 to produce the percentage columns above.  Also, it'd be best to not pre-round the ratios so that the calculations are more accurate (rounding the final output amounts/percentages here is fine).

Reply:
yup also realized this ... when I realized I forgot to update the tests *facepalm*